vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -142,28 +140,28 @@ The Entando Standard Banking Demo application demonstrates a number of the major
 
 ### Micro Frontends
 
-The application includes six micro frontends (MFEs) showcasing complementary features and custom functionality.
+The application includes six micro frontends (MFEs) showcasing complementary features and custom functionality. The key features of each are listed below.
 
 #### 1. Card MFE
 
 ![SeedCard.png](./images/SeedCard.png)
 
-- A React micro frontend placed on the My Dashboard page.
-- Makes an API call to the banking microservice to fetch a numeric result dependent on the current card selection. The displayed value updates with changes to the card configuration.
-- Authorization-aware and passes the bearer token to the microservice. If an unauthenticated user renders the dashboard page, the widget displays an error message.
-- Emits events that are consumed by the Transaction Table widget.
+- The Card MFE is a React micro frontend located on the My Dashboard page.
+- It makes an API call to the banking microservice to fetch a numeric result dependent on the current card selection. The displayed value updates with changes to the card configuration.
```

Comment:
>> Banking microservice

Reply:
lowercase, like any of our other features/functionalities that are also general concepts and not referenced in full, e.g. Entando Bundle vs bundle

Reply:
It's not a big deal but it's not a general concept because there is a microservice named banking as noted below in the microservice section.

Reply:
no, there is no microservice with the name Banking Microservice. there's a /banking path but that's different. like we use /auth for Keycloak but there's no Auth Service